backend/account/src/main/java/com/htec/account/controller/AuthorizationController.java

```diff
@@ -34,4 +32,9 @@ public ResponseEntity getUserById(@AuthenticationUser AuthUser authUser) {
         RefreshTokenResponseModel refreshTokenResponseModel = new RefreshTokenResponseModel(jwtToken);
         return ResponseEntity.status(HttpStatus.OK).body(refreshTokenResponseModel);
     }
+
+    @GetMapping("/authenticate")
+    public ResponseEntity authenticate(@AuthenticationUser AuthUser authUser) {
```

Comment:
you dont need @AuthenticationUser here, cause you are not using it

Reply:
Removed.